### PR TITLE
RHOARDOC-1583: Add tomcat starter as a dependency in the SB WAR file deployment procedure.

### PR DESCRIPTION
**Pull request by inoxx03**

### CODE REVIEW

File: docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc [hunk 1]
```diff
@@ -30,13 +30,34 @@ You must configure your build and deployment settings to ensure that your applic
...
 .Example `pom.xml`
 [source,xml,options="nowrap",subs="attributes+"]
 ----
+<project ...>
+  ...
+  <dependencies>
+    ...
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-tomcat</artifactId>
```
Comment (inoxx03) [inline docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc]: @liborfuka Would adding `spring-boot-starter-tomcat` as a dependency be  the correct way of specifying an application server?
Or would it be better to specify the productized JWS Tomcat artifact directly?

Comment (liborfuka) [reply docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc]: Its correct way to add tomcat as dependecny.

Comment (inoxx03) [reply docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc]: @liborfuka thank you

**Comment by tradej** [comment]
LGTM, merging.

**Pull request merged**